support-frontend/assets/components/personalDetails/personalDetails.tsx

```diff
@@ -1,18 +1,33 @@
 // ----- Imports ----- //
 import { css } from '@emotion/react';
-import { from, space, visuallyHidden } from '@guardian/source-foundations';
+import {
+	from,
+	headline,
+	space,
+	visuallyHidden,
+} from '@guardian/source-foundations';
 import { TextInput } from '@guardian/source-react-components';
 import type { ContributionType } from 'helpers/contributions';
 import { emailRegexPattern } from 'helpers/forms/formValidation';
 import type { PersonalDetailsState } from 'helpers/redux/checkout/personalDetails/state';
 
+const header = css`
+	${headline.xsmall({ fontWeight: 'bold' })};
+	margin-top: ${space[2]}px;
+	margin-bottom: ${space[4]}px;
+	${from.tablet} {
+		font-size: 28px;
+	}
+`;
+
 const hiddenHeading = css`
 	${visuallyHidden};
 `;
 
-const fieldGroupStyles = css`
+const fieldGroupStyles = (hideDetailsHeading?: boolean) => css`
 	position: relative;
 	margin-top: ${space[4]}px;
+	margin-top: ${hideDetailsHeading ? `${space[4]}px` : '0'};
```

Comment:
Is the margin-top needed on line above this one?

Reply:
good spot, I should have removed the top one ...